lib/pure/pathnorm.nim

```diff
@@ -100,15 +100,13 @@ proc addNormalizePath*(x: string; result: var string; state: var int;
   if result == "" and x != "": result = "."
 
 proc normalizePath*(path: string; dirSep = DirSep): string =
-  ## Example:
-  ##
-  ## .. code-block:: nim
-  ##   assert normalizePath("./foo//bar/../baz") == "foo/baz"
-  ##
-  ##
-  ## - Turns multiple slashes into single slashes.
-  ## - Resolves '/foo/../bar' to '/bar'.
-  ## - Removes './' from the path (but "foo/.." becomes ".")
+  runnableExamples:
+    when defined(posix):
+      # - Turns multiple slashes into single slashes.
+      # - Resolves '/foo/../bar' to '/bar'.
+      # - Removes './' from the path (but "foo/.." becomes ".")
+      doAssert normalizePath("./foo//bar/../baz") == "foo/baz"
+
```

Comment:
```suggestion
      doAssert normalizePath("./foo//bar/../baz") == "foo/baz"

## - Turns multiple slashes into single slashes.
## - Resolves `'/foo/../bar'` to `'/bar'`.
## - Removes `'./'` from the path, but `"foo/.."` becomes `"."`.
```

Theres no reason for the comments to be inside the code, and the code examples should be as minimal as possible I think.